UI/login.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="description" content="Affordable and professional web design">
+    <meta name="keywords" content="web design, stack overflow, question and answer">
+    <meta name="author" content="Eugene Owak">
+    <title>Stack Overflow-lite | Welcome</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="./css/style.css" />
+    <script src="main.js"></script>
+</head>
+
+<body>
+    <header>
+        <div class="container">
+            <div id="branding">
+                <h1 class="margin0;">Stack Overflow-lite</h1>
```

Comment:
Used inline style

Reply:
That's a class I created because I realized I needed no margins in many places, thus the name, which make it look like a style.

Reply:
Okay, I didn't see this one